assets/js/src/core/modules/asset/editor/image/tab-manager/tabs/properties/components/table/cells/boolean-cell/boolean-cell.tsx

```diff
@@ -0,0 +1,71 @@
+/**
+* Pimcore
+*
+* This source file is available under two different licenses:
+* - Pimcore Open Core License (POCL)
+* - Pimcore Commercial License (PCL)
+* Full copyright and license information is available in
+* LICENSE.md which is distributed with this source code.
+*
+*  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+*  @license    https://github.com/pimcore/studio-ui-bundle/blob/1.x/LICENSE.md POCL and PCL
+*/
+
+import { type DefaultCellProps } from '@Pimcore/components/grid/columns/default-cell'
+import React, { useEffect, useRef } from 'react'
+import { Checkbox, type CheckboxRef } from 'antd'
+import { useEditMode } from '@Pimcore/components/grid/edit-mode/use-edit-mode'
+import {
+  useStyles
+} from '@Pimcore/modules/asset/editor/image/tab-manager/tabs/properties/components/table/cells/boolean-cell/boolean-cell.styles'
+
+export const BooleanCell = (props: DefaultCellProps): React.JSX.Element => {
+  const { styles } = useStyles()
+  const { isInEditMode, disableEditMode, fireOnUpdateCellDataEvent } = useEditMode(props)
+  const checkboxRef = useRef<CheckboxRef>(null)
+
+  useEffect(() => {
+    if (isInEditMode) {
+      checkboxRef.current?.focus()
+    }
+  }, [isInEditMode])
+
+  function saveValue (): void {
+    fireOnUpdateCellDataEvent(checkboxRef.current!.input?.value ?? '')
+    // disableEditMode()
+  }
+
+  function getCellContent (): React.JSX.Element {
+    if (!isInEditMode) {
+      return (
+        <>
+          <Checkbox
+            checked={ props.getValue() }
+            disabled
+          />
+        </>
+      )
+    }
+
+    function onKeyDown (e: React.KeyboardEvent<HTMLInputElement>): void {
+      if (e.key === 'Escape') {
+        disableEditMode()
+      }
+    }
+
+    return (
+      <Checkbox
+        checked={ props.getValue() }
+        onChange={ saveValue }
```

Comment:
I would only send the update for the data when hitting enter.
Currently the value would directly send via update event even so when I try to abort with escape.

So maybe we use the original cell value just as an initial value for the component state. 
Hitting enter then sends the state value per update event.

Reply:
Not sure about the usability here. My idea was to send the changes each time you check the checkbox (because obv. it is a change). Don`t know if the "send on enter" conflicts with the keyboard navigation.